rosdep/python.yaml

```diff
@@ -6780,6 +6780,10 @@ python3-ifcfg:
   ubuntu:
     bionic: [python3-ifcfg]
     focal: [python3-ifcfg]
+python3-ignition-math6:
+  ubuntu:
+    focal: [python3-ignition-math6]
+    jammy: [python3-ignition-math6]
```

Comment:
```suggestion
    jammy: [python3-ignition-math6]
python3-ignition-gazebo6:
  ubuntu:
    focal: [python3-ignition-gazebo6]
    jammy: [python3-ignition-gazebo6]
```

Reply:
Good call, thanks!

Reply:
This introduced an error:
   ERR: list out of alphabetical order line 6787.  'python3-ignition-gazebo6' should come before 'python3-ignition-math6'

Reply:
fixed 🙃